docs/CLI.md

```diff
@@ -61,3 +61,12 @@ gulp has very few flags to know about. All other flags are for tasks to use if n
 
 **--log-level**, **-L**
     Set the loglevel. -L for least verbose and -LLLL for most verbose. -LLL is default.
+
+**--config**, **-c**
+    Manually set path of configuration file.
+
+**--ignore-config**
+    Ignore configuration files except files specified by --config flag.
+
+**--output-config**
```

Comment:
By this flag, users can check the current configurations and get the list of the customizable configurations. Since the configuration set is the result of merging multiple files, it would be useful to check the result. And it would be useful for sharing someone's configurations, for generating single configuration files from multiple configuration files by combining it with `--config` flag, and so on.